eth_tester/utils/backend_testing.py

```diff
@@ -221,6 +238,39 @@ def test_mine_multiple_blocks(self, eth_tester):
     #
     # Transaction Sending
     #
+    def test_send_raw_transaction_valid_raw_transaction(self, eth_tester):
+        # send funds to our sender
+        raw_privkey = b'\x11' * 32
+        test_key = keys.PrivateKey(raw_privkey)
+        eth_tester.send_transaction({
+            "from": eth_tester.get_accounts()[0],
+            "to": test_key.public_key.to_checksum_address(),
+            "gas": 21000,
+            "value": 1 * denoms.ether,
+        })
+        # create a raw transaction which spends previously the sent funds
+        unsigned_transaction = eth_tester.backend.chain.create_unsigned_transaction(
```

Comment:
This isn't going to work since these tests are designed to run across different backends so we can't reach down into the actual backend class to do things since each backend may be different.

What I suggest is hard coding the full signed raw transaction in this test so that you can bypass all of the signing logic.

Reply:
No problem. Make sense.
I have done it in the new commit.